redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/provision/RedbeamsProvisionEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.redbeams.flow.redbeams.provision;
+
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.redbeams.flow.redbeams.provision.event.allocate.AllocateDatabaseFailed;
+import com.sequenceiq.redbeams.flow.redbeams.provision.event.allocate.AllocateDatabaseSuccess;
+
+public enum RedbeamsProvisionEvent implements FlowEvent {
+    REDBEAMS_PROVISION_EVENT("REDBEAMS_PROVISION_EVENT"),
+    ALLOCATE_DATABASE_FINISHED_EVENT(EventSelectorUtil.selector(AllocateDatabaseSuccess.class)),
```

Comment:
I'm gonna be pedantic: This should be ALLOCATE_DATABASE_SERVER_FINISHED_EVENT, since it applies to database servers, not databases. Ditto for other enums, class names, etc. - I won't pollute my review with every instance of this.

Reply:
Agreed with the pedantry. :-) Will fix.